### PR TITLE
Updating comments

### PR DESCRIPTION
**Pull request by Sandipan-Ghosh**
**Fixed Issue**

Edited the comments of
* GetAllQuestions API of questions controller

* GetAllQuestionsAsync method of QuestionRepository

* getAllQuestions method of questions-dashboard.component

* getAllQuestions method of questions-service.ts

**Files Changed**

QuestionController.cs

QuestionRepository.cs

question-dashboard.component.ts

questions-services.ts

**Checks**

- [x] Naming Conventions
- [x] Server side code comments (proper English, grammar and no spelling mistakes)
- [x] Client side code comments (proper English, grammar and no spelling mistakes)
- [x] Unused variables, methods, blank spaces and name spaces. 
- [x] Optimal Code and code formatting
- [x] Commit History is proper, no merge is done. 
- [x] Commit/pull request messages should be proper to justify your feature
- [ ] All test cases are executed provided by tester.

**Known Issues**

 N/A

**Comments**

Only the comments of few methods are updated in this pull request.

### CODE REVIEW

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QuestionRepository.cs#L39](https://github.com/Promact/trappist/blob/571ac5d4bd4abe60e1237ad32972e87ebb8b862f/Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs#L39): Remove this commented out code. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public QuestionController(IQuestionRespository questionsRepository)
         }
 
         /// <summary>
-        /// Get All The Questions
+        /// Get all the questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs]: only first letter should be capital except for few cases.

File: Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ public QuestionRepository(TrappistDbContext dbContext)
         }
 
         /// <summary>
-        ///Get All Questions
+        ///Get all questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs]: same as above.

Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs]: No need to write function name ends with async.

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QuestionRepository.cs#L38](https://github.com/Promact/trappist/blob/b9d2bc3025f7430077ddae8641eb8df07fbff212/Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs#L38): Remove this commented out code. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs [hunk 3]
```diff
@@ -42,7 +42,7 @@ public QuestionController(IQuestionRespository questionsRepository)
         }
 
         /// <summary>
-        /// Get All The Questions
+        /// Get all the Questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs]: API to get all the Questions.

File: Trappist/src/Promact.Trappist.Repository/Questions/IQuestionRespository.cs [hunk 4]
```diff
@@ -20,7 +20,7 @@ public interface IQuestionRespository
         Task AddCodeSnippetQuestionAsync(QuestionAC questionAC);
 
         /// <summary>
-        /// Get all questions
+        /// Get all Questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Repository/Questions/IQuestionRespository.cs]: Method to get all the Questions.

File: Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs [hunk 5]
```diff
@@ -17,10 +17,9 @@ public QuestionRepository(TrappistDbContext dbContext)
         }
 
         /// <summary>
-        ///Get All Questions
+        ///Get all Questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs]: same as above.

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QuestionRepository.cs#L38](https://github.com/Promact/trappist/blob/ade780c88cc8ffebd15cdbd3a2cf4b2356620d77/Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs#L38): Remove this commented out code. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: Trappist/src/Promact.Trappist.Repository/Questions/IQuestionRespository.cs [hunk 6]
```diff
@@ -20,7 +20,7 @@ public interface IQuestionRespository
         Task AddCodeSnippetQuestionAsync(QuestionAC questionAC);
 
         /// <summary>
-        /// Get all questions
+        /// Method to get all Questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Repository/Questions/IQuestionRespository.cs]: Method to get all the Questions

File: Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs [hunk 7]
```diff
@@ -17,10 +17,9 @@ public QuestionRepository(TrappistDbContext dbContext)
         }
 
         /// <summary>
-        ///Get All Questions
+        ///Method to get all Questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs]: Same as above.

File: Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-dashboard/questions-dashboard.component.ts [hunk 8]
```diff
@@ -27,33 +27,33 @@ export class QuestionsDashboardComponent {
...
     }
-    //To Get All The Categories
+    //To get all the categories
     getAllCategories() {
         this.categoryService.getAllCategories().subscribe((CategoriesList) => {
             this.categoryArray = CategoriesList;
         });
     }
-    //To Get ALL Questions
+    //To get aLL questions
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-dashboard/questions-dashboard.component.ts]: what is this aLL?

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QuestionRepository.cs#L38](https://github.com/Promact/trappist/blob/c8ad2905701cfa87f4715d51ee24165692b1e91e/Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs#L38): Remove this commented out code. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Pull request merged**